lib/OpenQA/WebAPI/Controller/API/V1/User.pm

```diff
@@ -0,0 +1,53 @@
+package OpenQA::WebAPI::Controller::API::V1::User;
+use Mojo::Base 'Mojolicious::Controller';
+
+=pod
+
+=head1 NAME
+
+OpenQA::WebAPI::Controller::API::V1::User
+
+=head1 SYNOPSIS
+
+  use OpenQA::WebAPI::Controller::API::V1::User;
+
+=head1 DESCRIPTION
+
+Implements User API.
+
+=head1 METHODS
+
+=over 4
+
+=item delete()
+
+Post integer value to save User tour progress of current user in the database
+
+=back
+
+=cut
+
+sub delete {
+    my ($self)      = @_;
+    my $users       = $self->schema->resultset('Users');
+
+    if ($self->current_user->is_admin != 1){
+        return $self->render(json => {error => 'Only administrators could delete users'}, status => 401);
+    }
```

Comment:
We usually enforce this already within Mojolicious' routing. I don't think that the specific error message is worth the additional code here.

Reply:
I've committed a change. Is this what you talking about?

Reply:
@Martchus There are other two messages like that, for instance "not found" or "some error..". Is is what I found in the code in other API methods. Do you think I should remove these too?

Reply:
IMHO this is done